var/spack/repos/builtin/packages/openmc/package.py

```diff
@@ -18,12 +18,37 @@ class Openmc(CMakePackage):
     url = "https://github.com/openmc-dev/openmc/tarball/v0.10.0"
     git = "https://github.com/openmc-dev/openmc.git"
 
+    version('develop', branch='develop', submodules=True)
+    version('master', branch='master', submodules=True)
+    version('0.12.0', tag='v0.12.0', submodules=True)
```

Comment:
Why are we using a tag instead of a tarball for v0.12.0?

Reply:
@alalazo - OpenMC switched to using Git submodules starting with v0.12.0. GitHub doesnt store submodules in its default artifacts when you make a release. So v0.11.0 tarball is fine because no submodules, but v0.12.0 needs them so I went the tag route with submodules=True.